.github/workflows/deploy_images.yml

```diff
@@ -76,6 +76,7 @@ jobs:
           if ${{ inputs.orchestrator }} == 'true'; then GRADLE_TASKS+=":save-orchestrator:bootBuildImage "; fi
           if ${{ inputs.sandbox }} == 'true'; then GRADLE_TASKS+=":save-sandbox:bootBuildImage "; fi
           if ${{ inputs.preprocessor }} == 'true'; then GRADLE_TASKS+=":save-preprocessor:bootBuildImage "; fi
+          if ${{ inputs.demo }} == 'true'; then GRADLE_TASKS+=":save-demo:bootBuildImage "; fi
```

Comment:
Should be added to `workflow_dispatch.inputs` too, see the top of this file. Also there is default list of tasks for this workflow on line 65

Reply:
Done